tests/test_scanner.py

```diff
@@ -18,27 +18,26 @@
 # Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #
 import unittest
-from unittest.mock import Mock
+from unittest.mock import Mock, patch, call
 import threading
 from py_sonar_scanner.scanner import Scanner
 from py_sonar_scanner.configuration import Configuration
 
 
 class TestScanner(unittest.TestCase):
-
     def test_scanner_compute_command(self):
         cfg = Configuration()
         cfg.sonar_scanner_executable_path = "test"
         cfg.scan_arguments = ["a", "b"]
         scanner = Scanner(cfg)
 
-        assert scanner.compute_command() == ["test", "a", "b"]
+        self.assertEqual(scanner.compute_command(), ["test", "a", "b"])
```

Comment:
Now that I think about this, are we using `unittest` or `pytest` for testing?
I see we actually execute `pytest` to run our tests, yet the class structure and dedicated assertion is from `unittest` if I understand correctly.

Aren't we mixing things up here? Could it cause issues with test discovery? Maybe we should make things more consistent on that front (outside this PR and ticket though).

Reply:
It should not be an issue: https://docs.pytest.org/en/7.1.x/how-to/unittest.html pytest supports unittest out of the box. We could move everything to pytest that's a possibility, to be honest I don't have a strong opinion on that.

Reply:
That's good to know! I don't have a strong opinion either, we can decide later if we want to move to `pytest` or not.